### PR TITLE
Feature/styling instructor add course

### PR DESCRIPTION
**Pull request by MaryAngelique**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Loom Video

Please paste a link here of a quick loom video walking through what you did in your PR .

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/Classroom/StudentCard 2.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Card, Avatar, Badge } from 'antd';
+import { Link } from 'react-router-dom';
```
Comment (Adamh1990) [inline src/components/pages/Classroom/StudentCard 2.js:3]: Github is yelling about how { Link } isn't used

**Review by Adamh1990 — APPROVED** [review]
This looks amazing, well done! Other than a couple of unused variables and empty files (not sure if there could be plans for those later?) , good job

File: src/components/common/calendar/CalendarModal.js [hunk 2]
```diff
@@ -22,8 +22,7 @@ export default function CalendarModal(props) {
     form.resetFields();
   };
 
-  const onFinishFailed = errorInfo => {
-  };
+  const onFinishFailed = errorInfo => {};
```
Comment (moua0061) [inline src/components/common/calendar/CalendarModal.js:25]: This is a nice clean up. Looks more clean.

File: src/styles/InstructorStyles/addCourse.less [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+        font-weight: 300;
+    }
+
+    // Intro
+    .iac__intro {
+        font-size: @text__font-size--h5;
+        line-height: @text__line-height--h5;
+    }
+
+    // <---- Form Divs ---->
```
Comment (moua0061) [inline src/styles/InstructorStyles/addCourse.less:28]: I like that you comment the different sections of the form for easy to read for others.

**Review by moua0061 — APPROVED** [review]

**Review by ElchinNasirov — APPROVED** [review]

**Review by Raj-04 — APPROVED** [review]
Great Changes

**Pull request merged**